### PR TITLE
update html references

### PR DESCRIPTION
**Pull request by yitao-li**
Signed-off-by: Yitao Li <yitao@rstudio.com>

### CODE REVIEW

**Comment by falaki** [comment]
Databricks Connect tests failed. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2718-1/c2421f6e-8e9e-44cf-b455-f46a82486912.txt).

**Comment by falaki** [comment]
Databricks Connect tests failed. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2718-2/54e9c15f-d2c2-46f1-95b7-ace56ce40e56.txt).

File: docs/reference/index.html [hunk 1]
```diff
@@ -11,17 +11,6 @@ <h1 class = "content-header">Function Reference test</h1>
 
     <div class="doc-page-index">
     <ul data-gumshoe>
-      <li><a href="#section-spark-operations">Spark Operations</a></li>
```
Comment (yitao-li) [inline docs/reference/index.html:14]: @javierluraschi In your previous pull request to rebuild HTML documents, what was the command or the option to select in RStudio that would tell `pkgdown` to render all the section elements here?

For some reason my attempt to update sparklyr HTML reference would always cause all of those elements to disappear, and I'm trying to figure out why :sweat_smile: 

I would really like to document this in the sparklyr Github wiki page as well for future reference.

**Comment by falaki** [comment]
Databricks Connect tests failed. View logs [here](https://sparklyr-jenkins.s3.amazonaws.com/logs/jenkins-sparklyr-PR-2718-3/8f4dfcb0-1ea6-4531-bc5e-50eebc86d15e.txt).

File: docs/reference/index.html (hunk 1 above)
Comment (javierluraschi) [reply docs/reference/index.html:14]: I think we just need to run `pkgdown::build_site()` from `sparklyr` and then something else from `spark.rstudio.com`. Honestly, we should also consider recreating the website entirely, we keep the content, but rebuild it in whichever way you want. The LF AI at some point also suggested having the docs also under docs.sparklyr.ai so maybe something we can consider if the current process gets hard to reverse engineer.

**Comment by yitao-li** [comment]
I think unless there is some better alternative, we will have to replace this with the workaround in https://github.com/sparklyr/sparklyr/pull/2726 plus https://github.com/rstudio/spark.rstudio.com/pull/107 for the time-being.

**Pull request closed without merging**